### PR TITLE
OOP school library: decorate a class

### PR DESCRIPTION
**Pull request by kidde60**

### CODE REVIEW

File: decorator.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require_relative 'nameable'
+class Decorator < Nameable
```
Comment (GzimAsani) [inline decorator.rb:2]: - [ ]  Add `attr_accessor :nameable` so it can be getter and setter

File: capitalize.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+require_relative 'decorator'
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```
Comment (GzimAsani) [inline capitalize.rb]: - [ ] `.upcase` makes the whole word upcase where you need to capitalize only the first letter, so I would suggest you use `.capitalize` instead

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @kidde60            :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your classes :+1: 

 ## Required Changes 


- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @kidde60,

✔️ Implementation of class decoration

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**